docs/how-to/redeployment.md

```diff
@@ -0,0 +1,37 @@
+# Redeployment Guide
```

Comment:
I think this should be renamed as "Redploy". So the navigation reads as "How-to"-> "Redeploy"

Reply:
Do you mean the filename itself (currently `redeployment.md`) or the title here (or both)?

Reply:
Yes, indeed, updated, thanks!